rest/config_manager.go

```diff
@@ -768,9 +768,11 @@ func (b *bootstrapContext) computeMetadataID(ctx context.Context, registry *Gate
 	for _, cg := range registry.ConfigGroups {
 		for dbName, db := range cg.Databases {
 			if dbName == config.Name {
+				base.InfofCtx(ctx, base.KeyConfig, "Using metadata ID=%q from registry for db %q", base.MD(db.MetadataID), base.MD(dbName))
 				return db.MetadataID
 			}
 			if db.MetadataID == defaultMetadataID {
+				// do not return standardMetadataID here in case there was a different metadata ID assigned to this database
```

Comment:
Might be worth a (debug?) log here even though we're not returning, so we can log _which_ database has the default metadata ID.